src/components/Split/index.js

```diff
@@ -0,0 +1,40 @@
+import React, {PropTypes} from 'react'
+import Heading from 'components/Heading'
+
+const Split = ({
+  intro,
+  title,
+  main,
+  aside,
+}) => (
+  <div>
+
+    {intro}
+
+    <Heading level='1'>
+      {title}
+    </Heading>
+
+    <div className='flex-ns'>
+
+      <main className='mr4-ns w-50-ns'>
+        {main}
+      </main>
+
+      <aside className='w-50-ns mt5 mt0-ns'>
+        {aside}
+      </aside>
+
+    </div>
+
+  </div>
+)
+
+Split.propTypes = {
+  intro: PropTypes.node,
+  title: PropTypes.string.isRequired,
+  main: PropTypes.node.isRequired,
+  aside: PropTypes.node.isRequired,
+}
+
+export default Split
```

Comment:
This split component is used for most "screens" to give a responsive typical layout